app/scripts/views/pair/auth_allow.js

```diff
@@ -16,8 +17,25 @@ class PairAuthAllowView extends FormView {
     'click #cancel': preventDefaultThen('cancel')
   });
 
-  initialize () {
+  beforeRender () {
+    const account = this.getSignedInAccount();
+
     this.listenTo(this.broker, 'error', this.displayError);
+
+    if (! account) {
```

Comment:
@vladikoff do you want this here on in [pairing/index.js](https://github.com/mozilla/fxa-content-server/blob/2a3120429df99160b3620ae875b0985f524f875a/app/scripts/views/pair/index.js#L28) before the user clicks the button to pair?

Reply:
We should discuss this, we can block it here or on /pair. If it is on /pair then there is a sneaky way you can still pair with 2FA (by going to a specific about:prefs url)